README.md

```diff
@@ -44,25 +44,34 @@ For more information about the project please join our [Discord](https://discord
 ## Git branches and development
 The default branch in the Github repo is `development`. However, releases are cut from the `main` branch. In general, PRs should be made against the `development` branch and reviewed by at least one other person before being merged. When ready for a release, a PR should be made from `development` to `main` and reviewed. Once happy with the PR, it can be merged and then a [release can be drafted for distribution](#automatic-github-distribution).
 
-## Install
+## Contributing
```

Comment:
i'd name this `Installing & Contributing`, as people might want to run the app locally but not necessarily contribute.